utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java

```diff
@@ -106,10 +107,24 @@ public static void deploy() {
     }
 
     public static void cleanUp() {
-        OpenShiftUtils.getInstance().getDeploymentConfigs().stream().filter(dc -> dc.getMetadata().getName().equals(APP_NAME)).findFirst()
-                .ifPresent(dc -> OpenShiftUtils.getInstance().deleteDeploymentConfig(dc, true));
-        OpenShiftUtils.getInstance().getServices().stream().filter(service -> APP_NAME.equals(service.getMetadata().getName())).findFirst()
-                .ifPresent(service -> OpenShiftUtils.getInstance().deleteService(service));
-        TestUtils.sleepIgnoreInterrupt(5000);
+        try {
+            OpenShiftUtils.getInstance().getDeploymentConfigs().stream().filter(dc -> dc.getMetadata().getName().equals(APP_NAME)).findFirst()
+                    .ifPresent(dc -> OpenShiftUtils.getInstance().deleteDeploymentConfig(dc, true));
+            OpenShiftUtils.getInstance().getServices().stream().filter(service -> APP_NAME.equals(service.getMetadata().getName())).findFirst()
+                    .ifPresent(service -> OpenShiftUtils.getInstance().deleteService(service));
+            TestUtils.sleepIgnoreInterrupt(5000);
+        } catch (Exception e) {
+            log.error("Error thrown while trying to delete mysql database. It is just deletion, it should not affect following tests.");
+            log.error(e.toString());
```

Comment:
log.error("Error thrown while trying to delete mysql database. It is just deletion, it should not affect following tests.", e)?